addons/hr_timesheet/models/res_config_settings.py

```diff
@@ -1,21 +1,36 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     module_project_timesheet_synchro = fields.Boolean("Awesome Timesheet")
     module_project_timesheet_holidays = fields.Boolean("Record Time Off")
-    project_time_mode_id = fields.Many2one(
-        'uom.uom', related='company_id.project_time_mode_id', string='Project Time Unit', readonly=False,
+
+    @api.model
+    def _default_encoding_uom_id(self):
```

Comment:
This seems to be unused right ?

Reply:
I've added this method as a default on relevant fields